setup.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-vars */
+import 'dotenv/config';
+import { User } from './src/moduls/User.js';
+import { Token } from './src/moduls/Token.js';
+import { Room } from './src/moduls/Room.js';
+import { Messages } from './src/moduls/Messages.js';
```

Comment:
The import path for `Messages.js` is incorrect. The directory `moduls` should be `modules` as per the file structure.